tools/k8s-secret-syncer/resource_syncer.py

```diff
@@ -0,0 +1,89 @@
+import os
```

Comment:
I think this file requires a copyright header. The public style guide is 4-space indent even though internally we use 2-space.
You should also have a shebang line.

```
#!/usr/bin/env python2.7
```